spaceship.zsh

```diff
@@ -42,6 +42,7 @@ if [ -z "$SPACESHIP_PROMPT_ORDER" ]; then
   SPACESHIP_PROMPT_ORDER=(
     time          # Time stampts section
     user          # Username section
+    write_perm    # Write permission in current directory
```

Comment:
I think it makes more sense to show the lock icon after the directory name, not before.